src/routes/LPOptimisers/VaultFormRoute/VaultDepositForm/VaultDepositForm.tsx

```diff
@@ -48,11 +56,19 @@ export const VaultDepositForm: React.FunctionComponent<VaultDepositFormProps> =
         )
         .then(
           () => {
+            pushEvent(account ?? '', {
+              event: 'successful_tx',
+              eventValue: 'deposit',
```

Comment:
not sure what the goal is here but in the past we passed an event value for `successful_tx` following this pattern:
```
{
          notional: ...
          margin: ...,
          action: ....,
 }
```
Are we interested in how much the users deposited? If yes, maybe we should send an amount param

Reply:
fixed in: ec3398bc5c4b3b49b329f53f8f1f936b86c95abd